hcloud/server.go

```diff
@@ -134,14 +134,49 @@ func (c *ServerClient) Get(ctx context.Context, idOrName string) (*Server, *Resp
 	return c.GetByName(ctx, idOrName)
 }
 
+// ServerPage serves as accessor of the servers API pagination.
+type ServerPage struct {
+	Page
+	content []*Server
+}
+
+// Content contains the content of the current page.
+func (p *ServerPage) Content() []*Server {
```

Comment:
Since it’s a specialized struct anyway, what about naming it `Servers()` instead of `Content()`?

Reply:
I think its simpler for the user to use when every Page object has the same method names.